pkg/k8sports/kubernetes_helpers.go

```diff
@@ -1,3 +1,4 @@
+// Copyright 2015 The Kubernetes Authors.
 // Copyright (c) 2017 Intel Corporation
```

Comment:
`You must cause any modified files to carry prominent notices stating that You changed the files; and`